### DISCUSSION TITLE
I don't know why this happens and I don't know how to change it

### DISCUSSION THREAD

**Discussion by Lact3o** [Pryaxis/TShock#2908] [Q&A]
when someone tries to break blocks very fast on my server out of nowhere they get stuck in a web and can't do anything for a few seconds. The same thing happens if you have a weapon that does a lot of damage. I need help to know if that can be disabled.

**Comment by sgkoishi** [comment]
[ProjectileThreshold](https://github.com/Pryaxis/TShock/blob/general-devel/docs/config-file-descriptions.md#projectilethreshold)
[TileKillThreshold](https://github.com/Pryaxis/TShock/blob/general-devel/docs/config-file-descriptions.md#tilekillthreshold)
[MaxDamage](https://github.com/Pryaxis/TShock/blob/general-devel/docs/config-file-descriptions.md#maxdamage)
